test/provider/ChannelHandler.unittest.ts

```diff
@@ -144,12 +144,12 @@ describe('When setting the last broadcast context for a channel', () => {
     const testChannel = createMockChannel();
     const testContext = {type: 'test'};
 
-    let testWindow: AppWindow;
+    let testWindow: AppConnection;
```

Comment:
As general policy, should this be `testWindow`, or `testConnection`?

Reply:
IMO, should only use "window" if it's strictly only ever going to be an OpenFin window.

In this case, the test will only ever use `EntityType.WINDOW` - but the underlying code is intended to be used with connections as well as windows. A bit arbitrary, maybe - but updated this to `testConnection`.